std/stdio.d

```diff
@@ -2896,7 +2911,9 @@ See $(LREF byChunk) for an example.
     {
         import std.traits : hasIndirections;
     private:
-        FILE* fps;
+        // Access the FILE* handle through the 'file_' member
+        // to keep the object alive through refcounting
+        File file;
```

Comment:
I'd say you should be consistent with LockingTextWriter and use `file_` here instead of `file`, but either way, the comment doesn't match the member.

Reply:
Ok, thanks for the comments!
Fixing :)